biz.aQute.repository/src/aQute/maven/provider/POM.java

```diff
@@ -443,7 +457,7 @@ private void resolve(Dependency d) {
 	}
 
 	private void getDependencies(Map<Program, Dependency> deps, EnumSet<MavenScope> scope, boolean transitive,
-		Set<Program> visited) throws Exception {
+		Set<Program> visited, boolean useDependencyManagement) throws Exception {
```

Comment:
Name this `dependencyManagement` to match other places.

Reply:
This one I would keep, becuse we already have a field that represents the `dependencyManagement` of the pom.

Reply:
Same here